lab_1/main.py

```diff
@@ -3,6 +3,8 @@
 A concordance extraction
 """
 
+from os import path
+separators = '.,-!?:"@#<>&*%\n'
```

Comment:
If you do not use throughout several of your functions - place it where you use it.